src/Microsoft.ML.OnnxTransform.StaticPipe/DnnImageFeaturizerStaticExtensions.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.OnnxTransform.StaticPipe
+{
+    public static class DnnImageFeaturizerStaticExtensions
```

Comment:
>public [](start = 4, length = 6)

ideally would want to document, but i get this is low-pri.

Reply:
yeap. low-pri . 

not looking to modify documentation in this PR

---
In reply to: [242805885](https://github.com/dotnet/machinelearning/pull/1914#discussion_r242805885) [](ancestors = 242805885)